worker/src/lib/dbLogger/DBLoggable.ts

```diff
@@ -751,6 +751,7 @@ export class DBLoggable {
         lytixKey: requestHeaders.lytixKey ?? undefined,
         lytixHost: requestHeaders.lytixHost ?? undefined,
         posthogHost: requestHeaders.posthogHost ?? undefined,
+        sessionId: requestHeaders?.sessionHeaders?.sessionId ?? undefined,
```

Comment:
I wasn't sure if this makes more sense In `log.request`, but wasn't sure of the consequences of a change like that.